outstatic/content/exams/matematika-9-2023-1.md

```diff
@@ -71,4 +157,82 @@ Délka vycházkové trasy je přesně 6 km, což je trojnásobek délky přímé
 ## 11.2 Na mapě je vycházková trasa o 20 cm **delší** než přímá trasa.
 ## 11.3 Měřítko turistické mapy je 1 ∶ 200 000. 
 
----
\ No newline at end of file
+---
+
+**VÝCHOZÍ TEXT A OBRÁZEK K ÚLOZE 12**
+>Vnitřní prostor haly má tvar kvádru ABCDEFGH, jehož výška je 6 m a délka 15 m.
+Uvnitř haly je na podlaze, stropě a dvou stěnách vyznačena uzavřená lomená čára ACFHA.
+Úhlopříčka vyznačená na podlaze haly měří 17 m a tvoří úsek AC této lomené čáry.
+ 12 **Jaká je délka lomené čáry** ACFHA?
+ A) 46 m
+ B) 50 m
+ C) 54 m
+ D) 68 m
+ E) jiná délka
+
+---
+
+**VÝCHOZÍ TEXT K ÚLOZE 13**
+>Obsah pláště rotačního válce je třikrát větší než obsah jedné podstavy tohoto válce.
```

Comment:
blockquote should be followed by space
> Obsah ....

check all usage in document

Reply:
i dont give fuck